src/tests/unit/tests/DetailsView/components/__snapshots__/issue-filing-dialog.test.tsx.snap

```diff
@@ -28,32 +28,23 @@ exports[`IssueFilingDialog componentDidUpdate dialog is not open & props have ch
           "axeVersion": undefined,
           "browserSpec": undefined,
           "extensionVersion": undefined,
-          "getEnvironmentInfo": [Function],
```

Comment:
> "getEnvironmentInfo" [](start = 9, length = 21)

am not sure, why these properties are removed as part of upgrading to es2017

Reply:
so, does jest' jsdom also supports es2017 usages?

---
In reply to: [314827727](https://github.com/microsoft/accessibility-insights-web/pull/1081#discussion_r314827727) [](ancestors = 314827727)

Reply:
> why these properties are removed as part of upgrading to es2017
I investigated this for a bit but wasn't able to pinpoint the specific change that updated the proxy serialization behavior.

> does jest' jsdom also supports es2017 usages?
Yes (we have a huge number of unit tests that rely on this, which are still passing with this update)